test/fixtures/rules/std_rule_cases/L052.yml

```diff
@@ -464,3 +464,67 @@ test_fail_newline_create_table_inline_comment:
     rules:
       L052:
         multiline_newline: true
+
+test_fail_tsql_semicolon_after_each_statement_1:
+  fail_str: |
+    CREATE FUNCTION dbo.funxn (
+        @incoming varchar(50)
+    ) RETURNS int
+    AS
+    BEGIN
+        IF 1 = 1 SET @result = 1
+        RETURN @result
+    END
+  fix_str: |
+    CREATE FUNCTION dbo.funxn (
+        @incoming varchar(50)
+    ) RETURNS int
+    AS
+    BEGIN
+        IF 1 = 1 SET @result = 1;;
```

Comment:
@jpers36, @tunetheweb: Here, the rule is adding two semicolons because there are two nested statements ending at the same place: The `IF` statement and the `SET` statement inside it. This looks odd. What would we prefer to see here?